botfront/client/main.less

```diff
@@ -333,6 +333,11 @@ p.overflow {
     background: #fff none !important;
 }
 
+.read-only{
+    pointer-events: none;
+    cursor: not-allowed !important;
```

Comment:
this is a generalized class for blocking edits on a container so the context is unknown. the idea is if this class is added to any element it will block interactions.